src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Properties/Resources.resx

```diff
@@ -190,4 +190,16 @@
   <data name="wox_plugin_windowwalker_SettingKillProcessTree" xml:space="preserve">
     <value>Kill process and its child processes</value>
   </data>
+  <data name="wox_plugin_windowwalker_SettingExplorerSettingInfo_Description" xml:space="preserve">
+    <value>This message is only shown, when searching with direct activation command.</value>
+  </data>
+  <data name="wox_plugin_windowwalker_SettingKillProcessTree_Description" xml:space="preserve">
+    <value>Be careful when activating this.</value>
+  </data>
+  <data name="wox_plugin_windowwalker_SettingOpenAfterKillAndClose_Description" xml:space="preserve">
+    <value>This feature isn't working, if the kill process confirmation is enabled.</value>
+  </data>
+  <data name="wox_plugin_windowwalker_SettingSubtitleDesktopName_Description" xml:space="preserve">
+    <value>The name is only shown in the subtitle and tool tip, if more than one desktop exists.</value>
```

Comment:
"If there are multiple desktops the name will only be shown in the subtitle and tooltip."